src/documentation/programming-guide.md

```diff
@@ -464,9 +464,11 @@ tree, [2]
 
 Thus, `GroupByKey` represents a transform from a multimap (multiple keys to individual values) to a uni-map (unique keys to collections of values).
 
-`CoGroupByKey` is a `PTransform` based on `GroupByKey`, differs from `GroupByKey`, it merges multiple key/value `PCollection`s with the same key type, and emits a collection of `KV<K, CoGbkResult>` pairs. It's usually used as a JOIN operation in Beam pipeline(describe in [design-your-pipeline#multiple-sources]({{ site.baseurl }}/documentation/pipelines/design-your-pipeline/#multiple-sources)). 
+##### **Joins with CoGroupByKey**
 
-Given the input collections as below:
+`CoGroupByKey` joins two or more key/value `PCollection`s that have the same key type, and then emits a collection of `KV<K, CoGbkResult>` pairs. [design-your-pipeline#multiple-sources]({{ site.baseurl }}/documentation/pipelines/design-your-pipeline/#multiple-sources) shows an example pipeline that uses a join. 
```

Comment:
the link text inside the square brackets needs a small change to be Design Your Pipeline. it's not rendering correctly on the [staging site](http://apache-beam-website-pull-requests.storage.googleapis.com/195/documentation/programming-guide/index.html#transforms-gbk):
[Design Your Pipeline]({{ site.baseurl }}/documentation/pipelines/design-your-pipeline/#multiple-sources)